Makefile

```diff
@@ -107,6 +107,12 @@ build: # Build REANA client and cluster components.
 	pip uninstall -y reana-commons reana-client reana-cluster reana-db pytest-reana && \
 	reana-dev install-client && \
 	reana-dev install-cluster && \
+	if [ "${DEBUG}" -gt 0 ]; then \
+		cd ${PWD}/../reana-server/ && \
+		python setup.py bdist_egg && \
```

Comment:
Note that we have a line in "Example 3" that tells people they have to run pip install in reana-server.  If we do it for them, we can delete that line above :wink: